dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java

```diff
@@ -0,0 +1,146 @@
+package io.dockstore.client.cli;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static io.dockstore.client.cli.ArgumentUtility.CWL_STRING;
+import static io.dockstore.client.cli.ArgumentUtility.WDL_STRING;
+
+public class LaunchTestIT {
+    //create tests that will call client.checkEntryFile for workflow launch with different files and descriptor
+
+    public static String checkEntryFile(String ext, String content, String descriptor){
```

Comment:
These tests should really test AbstractEntryClient.testEntryFile rather than a test method that shares a similar structure. I would refactor the former so that it is testable and use the tests below on it.